pages/[...slug].vue

```diff
@@ -1,12 +1,34 @@
 <template>
   <div>
-    <Breadcrumbs v-if="page.breadcrumbs" />
-    <DrupalTabs v-if="page.local_tasks" :tabs="page.local_tasks" />
-    <component :is="renderCustomElements(page.content)" />
+    <NuxtLayout :name="layout">
+      <main>
+        <SiteBreadcrumbs />
+        <DrupalTabs v-if="page.local_tasks" :tabs="page.local_tasks" />
+        <component :is="renderCustomElements(page.content)" />
+      </main>
+    </NuxtLayout>
   </div>
 </template>
 
 <script lang="ts" setup>
-const { fetchPage, renderCustomElements } = useDrupalCe()
-const page = await fetchPage(useRoute().path, { query: useRoute().query })
+const { fetchPage, renderCustomElements } = useDrupalCe();
+const page = await fetchPage(useRoute().path, { query: useRoute().query });
+// Set to false to support custom layouts, using <NuxtLayout> instead.
+definePageMeta({
+  layout: false,
+});
+const layout = computed(() => {
+  return page.value.page_layout || "default";
+});
+useHead({
+  title: page.value.title,
+  meta: page.value.metatags.meta,
+  link: page.value.metatags.link,
+  script: [
+    {
+      type: "application/ld+json",
+      children: JSON.stringify(page.value.metatags.jsonld || [], null, ""),
```

Comment:
also this looks not very nice, do we require the last two stringify parameters? this looks much simpler here https://stackoverflow.com/a/70609860

Reply:
removed last two stringify parameters